go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/nobl9/terraform-provider-nobl9
 go 1.16
 
 require (
-	github.com/google/uuid v1.1.2
+	github.com/google/uuid v1.3.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.7.0
-	github.com/nobl9/nobl9-go v0.0.0-20220506132101-a0e9c676ada0 // indirect
+	github.com/nobl9/nobl9-go v0.0.0-20220629130434-044750c773a8
```

Comment:
I think we should tag github.com/nobl9/nobl9-go with something like v0.1.0 and use it here. My reasoning is that we want to actually start tracking changes in the SDK with n9 releases.
The same in `nobl9/n9` repo.

Reply:
I was thinking about creating a tag once we go 'public', but I am willing to create one that follows our n9 version - v1.33.0.

Reply:
I just realized my message is not really an answer :D I am in favour of tagging it now with 1.33

Reply:
Great :D here it is! https://github.com/nobl9/terraform-provider-nobl9/pull/30/commits/7ddffba7c52d7db26f490334a26cf20b8d4e9983